OpenSim/Common/FileAdapter.h

```diff
@@ -175,7 +175,7 @@ class TableMissingHeader : public Exception {
 };
 
 /** FileAdapter is a DataAdapter that reads and writes files with methods
-readFile and writeFile respectively.                                          */
+read and writeFile respectively.                                          */
```

Comment:
Since `read()` isn't defined here, it may be confusing to users that they won't find the function prototype here. Could be useful to spend more words in the documentation here to explain how this works (e.g., read() is inherited from `DataAdapter`, child classes implement `extendRead()`, etc.). The suggestions are just rough ideas, no need to implement all of them necessarily. Also, if you think doxygen will be sufficient at picking this up, no need to add extra information either.

Reply:
Clarified.